examples/banananation/build.gradle

```diff
@@ -13,15 +13,24 @@ test {
   useJUnitPlatform()
 }
 
+jar {
```

Comment:
Reminder: Add a comment @Twisol

Reply:
Thanks; I've added a brief explainer comment to `banananation/build.gradle` about how models need to bundle their dependencies in general.